### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by pscriptos** [h1ddenpr0cess20/ollamarama-matrix#34] [General]
I just wanted to tell you that I thank you for your script.
I forked it tonight and moved it to a Docker container.

It's a lot of fun to use.

https://git.techniverse.net/scriptos/ollamarama-matrix-chatbot

**Comment by h1ddenpr0cess20** [comment]
Glad to hear that.  Sounds like you have a lot of the same interests based on your profile, so it was made with people like you in mind.  Homer was one of my favorite characters to test it with early on.

**Comment by pscriptos** [comment]
Homer is also my favourite character

I also have a SerbviceBot, which is Homer.
However, I already use it for other things like Uptime-Kuma.
That's why I used the smart arse from Simpsons, Professor Frink for this Script.

Thank you very much!
